src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.cs

```diff
@@ -53,6 +51,16 @@ internal class UnixPwmChannel : PwmChannel
             SetFrequency(frequency, dutyCycle, currentDutyCycleNs);
         }
 
+        /// <summary>The PWM chip number.</summary>
+        protected int Chip { get; }
```

Comment:
Why do we need them to be properties? We could keep them regular members and just change their visibility from private to protected right?

Reply:
Sure. I think that would work.